vcd/resource_vcd_vapp_vm.go

```diff
@@ -1943,36 +2087,115 @@ func setGuestProperties(d *schema.ResourceData, properties *types.ProductSection
 // based on the known ID of object.
 //
 // Example resource name (_resource_name_): vcd_vapp_vm.VM_name
-// Example import path (_the_id_string_): org-name.vdc-name.vapp-name.vm-name
+// Example import path for VM within vApp (_the_id_string_): org-name.vdc-name.vapp-name.vm-name
+// Example import path for standalone VM (_the_id_string_): org-name.vdc-name.vm-name
```

Comment:
I think we're missing an example with ID here too.

Reply:
added